.github/workflows/android_arm64.yml

```diff
@@ -116,7 +116,7 @@ jobs:
           apt --assume-yes install ccache scons ninja-build build-essential python3-pip
 
           # vcpkg requires cmake 3.19 or later
-          python3 -m pip install -U pip cmake
+          python3 -m pip install -U pip cmake~=3.28.0
```

Comment:
maybe we can install cmake from apt? and replace ubuntu20 => ubuntu22 to have 3.19 and newer

Reply:
I'm not sure why it was done via pip initially (@akashchi, maybe you can remember?) 
I suggest merging this one first to unblock the master, and then we'll see how to improve it